fixated.js

```diff
@@ -12,18 +15,19 @@ function Fixated(format, opt) {
   opt.writableObjectMode = true;
   opt.readableObjectMode = false;
   opt.fillChar = opt.fillChar || ' ';
+  this._rowDelimiter  = (rowDelimiters[opt.rowDelimiter] ? rowDelimiters[opt.rowDelimiter] : rowDelimiters.windows);
 
```

Comment:
How about:
```
his._rowDelimiter  = opt.rowDelimiter || require('os').EOL;
```
It allows to insert any delimiter from the outside and it falls back to the current operating system default